### PR TITLE
Rename 'pageId' term to 'windowId' (close #4586)

### LINKED ISSUE #4586: [Multi browser window] Refactor pageId to windowId
IMHO `pageId` naming is not very self-describing. Actually, it's not a page ID, but a window ID, In some context it can cause confusion. E.g. when handling `t.getBrowserConsoleMessages`, console messages are recorded not for a page, but for a window (which can host a number of different pages during a test session).

### PR DESCRIPTION
**Pull request by miherlosev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
     "source-map-support": "^0.5.16",
     "strip-bom": "^2.0.0",
     "testcafe-browser-tools": "2.0.5",
-    "testcafe-hammerhead": "15.0.0",
+    "testcafe-hammerhead": "https://github.com/miherlosev/tmp/files/4000342/testcafe-hammerhead-16.0.0.zip",
```
Comment (AndreyBelym) [inline package.json]: Just a reminder: don't forget to replace it.

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by Farfurix** [comment]
@miherlosev I'll restart it

**Pull request merged**